stl/STL.natvis

```diff
@@ -0,0 +1,1596 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">

```

Comment:
Is 2010 the latest version? It's a good opportunity to update this file to the latest.

Reply:
That file is the schema I don't think we need to update it.